db.js

```diff
@@ -292,12 +297,16 @@ export class ActivityPubDB extends EventTarget {
       return
     }
 
-    // TODO: Error if no first page
     let next = collection.first
```

Comment:
If we're doing reverse order we should start with the last page and go go to the `previous` page instead of the `next` page